client/scripts/rails_only.jsx

```diff
@@ -8,3 +8,7 @@ require('es5-shim/es5-sham');
 // Due to issue: https://github.com/ariya/phantomjs/issues/12401
 // Phantomjs does not like promises
 require('es6-promise').polyfill();
+
+require('expose?jQuery!jquery');
+require('expose?$!jquery');
+require('expose?jquery-ujs!jquery-ujs');
```

Comment:
Ok, this is a entry point. How do you invoked it? I was just following the example from expose-loader, and thought it applied to the expose npm modules like jquery-ujs? 

It would be helpful if you provided a example of proper implementation?